nixos/modules/services/networking/openvpn.nix

```diff
@@ -53,11 +53,15 @@ let
         ${optionalString (
           cfg.down != "" || cfg.updateResolvConf
         ) "down ${pkgs.writeShellScript "openvpn-${name}-down" downScript}"}
-        ${optionalString (cfg.authUserPass != null)
-          "auth-user-pass ${pkgs.writeText "openvpn-credentials-${name}" ''
-            ${cfg.authUserPass.username}
-            ${cfg.authUserPass.password}
-          ''}"
+        ${lib.optionalString (cfg.authUserPass != null) (
```

Comment:
You can drop the lib here and below since there's (sadly) a with lib at the top. (In newer PRs, we'd just import what we need from lib). But this is not a merge blocker.